### ISSUE TITLE
Diabetes prediction  using machine learning models and visualization

### ISSUE THREAD

**Issue by RajeshJampa** [kritikaparmar-programmer/HealthCheck#98]
i found a dataset on kaggle about diabetes prediction by their symptoms which has 2k data points,by using machine learning models in one of the notebook they got 98.3% accuracy,I want to work on that and I will try to increase the accuracy by feature engineering and data cleaning,kindly assign this issue for me

**Comment by utkarsh0702** [comment]
@RajeshJampa 98.3% is the training accuracy, what about test accuracy and validation accuracy?

**Comment by RajeshJampa** [comment]
@utkarsh0702 98.3% roc_auc_score on the test set only sir

**Comment by utkarsh0702** [comment]
@RajeshJampa
